psycop/common/sequence_models/tests/test_embedding.py

```diff
@@ -54,19 +54,25 @@ def test_diagnosis_mapping(
             timestamp=dt.datetime(2021, 1, 1),
             value="A00",
             source_type="diagnosis",
-            source_subtype="test_name",
+            source_subtype="A",
         ),
         TemporalEvent(
             timestamp=dt.datetime(2021, 1, 3),
             value="A30",
             source_type="diagnosis",
-            source_subtype="test_name",
+            source_subtype="A",
+        ),
+        TemporalEvent(
+            timestamp=dt.datetime(2021, 1, 3),
+            value="A30",
+            source_type="diagnosis",
+            source_subtype="B",
```

Comment:
Would propose adding comments to each of these TemporalEvents, documenting why they're included. 

E.g. I assume this example is included to test that subtype = "B" is excluded? :-)